run_checks.php

```diff
@@ -151,6 +159,20 @@
     exit(1);
 }
 
+// Check that old events are still match expected format
+try {
+    list($result_old) = $client->searchEvents(1, start: $start->getTimestamp() * 1000, end: $end->getTimestamp() * 1000, reverse: true);
+    $identification_data_old = $result_old->getEvents()[0]->getProducts()->getIdentification()->getData();
```

Comment:
We could check `$result_old` count first, as we do in the `searchEvents` example above to make sure that `$result_old->getEvents()[0]` below won't throw unexpectedly.